tools/api-markdown-documenter/src/documentation-domain/ParagraphNode.ts

```diff
@@ -68,24 +68,4 @@ export class ParagraphNode
 	public static createFromPlainText(text: string): ParagraphNode {
 		return new ParagraphNode(createNodesFromPlainText(text));
 	}
-
-	/**
```

Comment:
Unused, and fundamentally invalid (adjacent paragraphs must be separated by blank lines, which this logic removes).